src/server/controllers/init.js

```diff
@@ -0,0 +1,275 @@
+                                        name: tag,
+                                    },
+                                    create: {
+                                        name: tag,
+                                    },
+                                },
+                            },
+                        };
+                    }),
+                },
+                categories: {
+                    create: fakedPost.categories.map((category) => {
+                        return {
+                            category: {
+                                connectOrCreate: {
+                                    where: {
+                                        name: category,
+                                    },
+                                    create: {
+                                        name: category,
+                                    },
+                                },
+                            },
+                        };
+                    }),
+                },
+            },
+            include: {
+                tags: {
+                    include: {
+                        tag: true,
+                    },
+                },
+            },
+        });
+        console.log('Created Post:', createdPost);
+
+        if (!createdPost) {
+            return res.status(500).json({ error: SERVER_ERROR_MESSAGE.INTERNAL_SERVER });
+        }
+    }
+};
+
+const seedComments = async (req, res) => {
+    for (let i = 0; i < NUMBER_OF_COMMENTS_TO_GENERATE; i++) {
+        const fakedComment = fakeComment();
+
+        const randomUserId = generateRandomInt(NUMBER_OF_USERS_AND_PROFILES_TO_GENERATE);
+
+        const randomPostId = generateRandomInt(NUMBER_OF_POSTS_TO_GENERATE);
+
+        const createdComment = await prisma.comment.create({
+            data: {
+                ...fakedComment,
+                postId: randomPostId,
+                userId: randomUserId,
+            },
+        });
+        console.log('Created Comment:', createdComment);
+
+        if (!createdComment) {
+            return res.status(500).json({ error: SERVER_ERROR_MESSAGE.INTERNAL_SERVER });
+        }
+    }
+
+    for (let i = 0; i < NUMBER_OF_COMMENTS_WITH_PARENT_TO_GENERATE; i++) {
+        const fakedCommentWithParent = fakeCommentWithParent();
+
+        const randomUserId = generateRandomInt(NUMBER_OF_USERS_AND_PROFILES_TO_GENERATE);
+
+        const randomPostId = generateRandomInt(NUMBER_OF_POSTS_TO_GENERATE);
+
+        const createdCommentWithParent = await prisma.comment.create({
+            data: {
+                ...fakedCommentWithParent,
+                postId: randomPostId,
+                userId: randomUserId,
+            },
+        });
+
+        console.log('Created Comment with Parent:', createdCommentWithParent);
+
+        if (!createdCommentWithParent) {
+            return res.status(500).json({ error: SERVER_ERROR_MESSAGE.INTERNAL_SERVER });
```

Comment:
Your "enums" could be extended a bit so you don't have to hard code the status codes:

```js
const SERVER_ERROR = {
  INTERNAL: { MESSAGE: 'Some message', CODE: 500 }
}

return res.status(SERVER_ERROR.INTERNAL.CODE).json({ error: SERVER_ERROR.INTERNAL.MESSAGE });
```

Reply:
So @alexamaingard did want to do it that way and I didn't. I thought it was easier for me to see the codes returned to the client if they were hard coded rather than having a string/variable. Pls don't tell Alexa 🤣